Don't require too much cmake. Python version check is already patched to find
the right version.

--- CMakeLists.txt~	2025-05-30 14:12:03.000000000 +0200
+++ CMakeLists.txt	2025-07-30 20:35:40.959067407 +0200
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.20)
+cmake_minimum_required(VERSION 3.12)
 
 project(rcutils)
 
